src/examples/solana_list_rewards.ts

```diff
@@ -0,0 +1,18 @@
+import { StakingReward } from "../coinbase/staking_reward";
+import { Coinbase } from "../coinbase/coinbase";
+import { Address } from "../coinbase/address";
+
+async function printStakingInfo() {
+  Coinbase.configureFromJson({ filePath: "~/Downloads/cdp_api_key.json" });
+
+  const startTime = new Date(2024, 5).toISOString();
+
+  const rewards = await StakingReward.list("solana-mainnet", Coinbase.assets.Sol, ["beefKGBWeSpHzYBHZXwp5So7wdQGX6mu4ZHCsH3uTar"], startTime, new Date().toISOString());
```

Comment:
Wondering if we can replace the network with a constant as well?
[I see solana has one too now](https://github.com/coinbase/coinbase-sdk-nodejs/blob/master/src/client/api.ts#L1676).